spotbugs-tests/src/test/java/edu/umd/cs/findbugs/LoadMessagesTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * FindBugs - Find bugs in Java programs
```

Comment:
New files should reference SpotBugs not FindBugs, and the copyright notice should not mention the University of Maryland.